pkg/node/node.go

```diff
@@ -809,13 +809,15 @@ func (s *Service) NodeExpandVolume(ctx context.Context, req *csi.NodeExpandVolum
 	}
 	log.WithFields(f).Info("Calling resize the file system")
 
-	// Rescan the device for the volume expanded on the array
-	for _, device := range devMnt.DeviceNames {
-		devicePath := sysBlock + device
-		err = s.Fs.GetUtil().DeviceRescan(context.Background(), devicePath)
-		if err != nil {
-			log.Errorf("Failed to rescan device (%s) with error (%s)", devicePath, err.Error())
-			return nil, status.Error(codes.Internal, err.Error())
+	if s.useFC || s.useISCSI {
+		// Rescan the device for the volume expanded on the array
+		for _, device := range devMnt.DeviceNames {
+			devicePath := sysBlock + device
+			err = s.Fs.GetUtil().DeviceRescan(context.Background(), devicePath)
+			if err != nil {
```

Comment:
can you remove some extra lines here.

Reply:
only the if condition has been added to the previous code (if s.useFC || s.useISCSI )